### ISSUE TITLE
'[]' passed as a default value for parameter

### ISSUE THREAD

**Issue by tsimafeip** [sqlalchemy/sqlalchemy#6915]
### Describe the bug

We are overloading in our code 'visit_table' method from SchemaDropper class (sql/ddl.py). SQLAlchemy 1.4 introduces new default parameter for the method - '_ignore_sequences' which is defaulted to '[]', which is completely incorrect.

### To Reproduce

```python
https://github.com/sqlalchemy/sqlalchemy/blob/fb81f9c8d914f9911925dd3f4e77d7fc374b267c/lib/sqlalchemy/sql/ddl.py#L1084
```

### Error

.    def visit_table(
        self,
        table,
        drop_ok=False,
        _is_metadata_operation=False,
        _ignore_sequences=[],
    ):

### Versions

- OS: Any
- Python: Any
- SQLAlchemy: 1.4.21
- Database: Any
- DBAPI: Any

**Comment by CaselIT** [comment]
Hi,

The issue is that `[]` is mutable, right? Maybe defaulting to `()` instead could be an option?

**Comment by tsimafeip** [comment]
Hello,

Yep, mutability is an issue. () will definitely help, 'None' defaulting is just more common practice. I will tweak my PR.

**Comment by zzzeek** [comment]
im not sure how easy /necessary it is for a test for this one.  just default it to tuple, this is very common practice within the internals (we in fact have an immutabledict object that is heavily used in this way as well).

**Comment by CaselIT** [comment]
actually I found only 2 other instances searching for `[]` and `{}`

**Comment by sqla-tester** [comment]
**tsimafeip** has proposed a fix for this issue in the **master** branch:

**Fixes: #6915** https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3022
